packages/extract-react-types/__snapshots__/test.js.snap

```diff
@@ -3889,6 +3889,54 @@ Object {
 }
 `;
 
+exports[`ts as expression 1`] = `
```

Comment:
This snapshot is not entirely correct and I'm not sure why 🤔

![image](https://user-images.githubusercontent.com/3030010/60950836-45259680-a33b-11e9-9704-cbb18900bbec.png)